File: CS/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraReports.UI;

namespace RepAlignCheckBoxTableCell
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            XtraReport1 report = new XtraReport1();

            report.DataSource = ManualDataSet.CreateData();

            new ReportPrintTool(report).ShowPreviewDialog();
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms

Namespace RepAlignCheckBoxTableCell
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Dim report As New XtraReport1()

			report.DataSource = ManualDataSet.CreateData()
			'report.ShowDesignerDialog();
			report.ShowPreviewDialog()
		End Sub
	End Class
End Namespace